components/Footer.js

```diff
@@ -35,170 +37,386 @@ const SocialLink = styled.a`
-                      values={{
-                        crowdinLink: (
-                          <StyledLink href="https://crowdin.com/project/opencollective" openInNewTab>
-                            Crowdin
-                          </StyledLink>
-                        ),
-                      }}
-                    />
-                  )}
-                >
-                  <InfoCircle size={16} />
-                </StyledTooltip>
-              </P>
-              <div data-cy="language-switcher">
-                {Object.keys(languages).map(key => (
-                  <div key={key}>
-                    <a
-                      title={languages[key].name}
-                      href={`/?language=${key}&set=1`}
-                      onClick={e => switchLanguage(e, key)}
-                    >
-                      {languages[key].nativeName || languages[key].name}
-                    </a>{' '}
-                    {languages[key].completion && <span>({`${languages[key].completion}`})</span>}
-                  </div>
-                ))}
-              </div>
+          </Container>
+          <Container
+            order={[null, null, null, null, '3']}
+            color="#6E747A"
+            textAlign={'left'}
+            mt={[4, null, null, null, 0]}
+            display={[null, 'none', null, null, 'block']}
+          >
+            <P as="div" pb={2} pt={2}>
+              <TranslateIcon />
+              <Span
+                mx={2}
+                style={{ verticalAlign: 'middle' }}
+                fontSize="10px"
+                fontWeight="600"
+                color="black.800"
+                lineHeight="15px"
+                letterSpacing="0.8px"
+                textTransform="uppercase"
+              >
+                <FormattedMessage id="footer.changeLanguage" defaultMessage="change language" />
+              </Span>
+              <StyledTooltip
+                content={() => (
+                  <FormattedMessage
+                    id="Footer.Languages.JoinEffort"
+                    defaultMessage="No technical skill is required to contribute to translations. You can join the effort on {crowdinLink} 🌐"
+                    values={{
+                      crowdinLink: (
+                        <StyledLink href="https://crowdin.com/project/opencollective" openInNewTab>
+                          Crowdin
+                        </StyledLink>
+                      ),
+                    }}
+                  />
+                )}
+              >
+                <InfoCircle size={16} />
+              </StyledTooltip>
+            </P>
+            <Container width={['230px']} my={2}>
+              <StyledSelect
+                options={languageOptions}
+                onChange={({ value }) => switchLanguage(value)}
+                defaultValue={languageOptions[0]}
```

Comment:
Should be the current locale

```suggestion
                defaultValue={intl.locale}
```